ansible_collections/f5networks/f5_modules/plugins/modules/bigip_device_group.py

```diff
@@ -334,6 +377,14 @@ def auto_sync(self):
             return 'yes'
         return 'no'
 
+    @property
+    def asm_sync(self):
+        if self._values['asm_sync'] is None:
+            return None
+        elif self._values['asm_sync'] == 'enabled':
+            return 'yes'
+        return 'no'
+
```

Comment:
remove

Reply:
Removing this would lead to the module not detecting whether the setting needs to be changed, therefore it is required (again, same as for `auto_sync`).